habitat/utils/gym_adapter.py

```diff
@@ -89,13 +89,11 @@ class HabGymWrapper(gym.Env):
     """
 
     def __init__(self, env, save_orig_obs: bool = False):
```

Comment:
Really minor nit: cache env.config.GYM object. A lot of redundant accesses in the new init.